From 6a67a2192ea2250ef8812167f739f98517ae99e8 Mon Sep 17 00:00:00 2001
From: liangshuang <liangshuang@eswincomputing.com>
Date: Fri, 12 Jul 2024 09:57:46 +0800
Subject: [PATCH 1/2] WIN2030-15838:fix:fix the issue of memalign fail.

Changelogs:
1.Change the size of the malloc area from 8m to 256m.

Change-Id: I89f345e5b7f1207f0cdb3ea67f7237c67bc0333d
---
 configs/eic7700_evb_defconfig         | 2 +-
 configs/hifive_premier_p550_defconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configs/eic7700_evb_defconfig b/configs/eic7700_evb_defconfig
index d4ecb735..f9e1426b 100644
--- a/configs/eic7700_evb_defconfig
+++ b/configs/eic7700_evb_defconfig
@@ -1,5 +1,5 @@
 CONFIG_RISCV=y
-CONFIG_SYS_MALLOC_LEN=0x800000
+CONFIG_SYS_MALLOC_LEN=0x10000000
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0xf0000000
diff --git a/configs/hifive_premier_p550_defconfig b/configs/hifive_premier_p550_defconfig
index 4543d557..6d371930 100644
--- a/configs/hifive_premier_p550_defconfig
+++ b/configs/hifive_premier_p550_defconfig
@@ -1,5 +1,5 @@
 CONFIG_RISCV=y
-CONFIG_SYS_MALLOC_LEN=0x800000
+CONFIG_SYS_MALLOC_LEN=0x10000000
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0xf0000000
-- 
2.17.1

